src/sagas/index.js

```diff
@@ -0,0 +1,10 @@
+import { all } from 'redux-saga/effects'
+import {loginFlow} from './loginSaga';
+import {watchFetchEmployees} from './employeeSaga/fetch';
+
+export default function* rootSaga() {
+  yield all([
+    loginFlow(),
+    watchFetchEmployees()
```

Comment:
Where are we using this file?

Reply:
I was earlier using this file in the store, but i removed its reference. But forgot to remove the file.